common-lib/src/main/scala/com/gu/mediaservice/model/ThrallMessage.scala

```diff
@@ -0,0 +1,89 @@
+package com.gu.mediaservice.model
+
+ import com.gu.mediaservice.lib.logging.{GridLogging, LogMarker}
+ import com.gu.mediaservice.model.leases.MediaLease
+ import com.gu.mediaservice.model.usage.UsageNotice
+ import org.joda.time.{DateTime, DateTimeZone}
```

Comment:
Do we still need to use joda time even though it is deprecated?

Reply:
I'm not sure, I just didn't want to alter the types from the existing message. (Aside from removing `Option`)

Reply:
probably best to move to java.time in a sep. PR